README.md

```diff
@@ -8,9 +8,9 @@ maturing](https://img.shields.io/badge/lifecycle-maturing-blue.svg)](https://lif
 [![CRAN
 status](https://www.r-pkg.org/badges/version/connectapi)](https://cran.r-project.org/package=connectapi)
 [![Codecov test
-coverage](https://codecov.io/gh/rstudio/connectapi/branch/main/graph/badge.svg)](https://app.codecov.io/gh/rstudio/connectapi?branch=main)
+coverage](https://codecov.io/gh/posit-dev/connectapi/branch/main/graph/badge.svg)](https://app.codecov.io/gh/posit-dev/connectapi?branch=main)
```

Comment:
Same as above. CRAN will flag this as a bad link if we release with this in like this. (Though I've generally been removing the badge links from the readme when I submit packages to CRAN, it's not helpful on their version of the docs and they so often lead to URL check failures.)